packages/camera/camera_avfoundation/example/ios/RunnerTests/AvailableCamerasTest.m

```diff
@@ -73,6 +72,80 @@ - (void)testAvailableCamerasShouldReturnAllCamerasOnMultiCameraIPhone {
     XCTAssertTrue([dictionaryResult count] == 3);
   }
 }
+- (void)testAvailableCamerasShouldReturnExternalCameraIfAvailable {
+  CameraPlugin *camera = [[CameraPlugin alloc] initWithRegistry:nil messenger:nil];
+  XCTestExpectation *expectation =
+      [[XCTestExpectation alloc] initWithDescription:@"Result finished"];
+
+  // iPhone 13 Cameras:
+  AVCaptureDevice *wideAngleCamera = OCMClassMock([AVCaptureDevice class]);
+  OCMStub([wideAngleCamera uniqueID]).andReturn(@"0");
+  OCMStub([wideAngleCamera position]).andReturn(AVCaptureDevicePositionBack);
+
+  AVCaptureDevice *frontFacingCamera = OCMClassMock([AVCaptureDevice class]);
+  OCMStub([frontFacingCamera uniqueID]).andReturn(@"1");
+  OCMStub([frontFacingCamera position]).andReturn(AVCaptureDevicePositionFront);
+
+  AVCaptureDevice *ultraWideCamera = OCMClassMock([AVCaptureDevice class]);
+  OCMStub([ultraWideCamera uniqueID]).andReturn(@"2");
+  OCMStub([ultraWideCamera position]).andReturn(AVCaptureDevicePositionBack);
+
+  AVCaptureDevice *telephotoCamera = OCMClassMock([AVCaptureDevice class]);
+  OCMStub([telephotoCamera uniqueID]).andReturn(@"3");
+  OCMStub([telephotoCamera position]).andReturn(AVCaptureDevicePositionBack);
+
+  // iPad External Camera
```

Comment:
is this external cameras for ipad only?

Reply:
Yes external cameras are only supported on iPadOS 17.0 for now

Reply:
in case of confusion, can you add a comment saying "iPhone doesn't support external camera"? or remove the iPad here?